iroha-cli/impl/grpc_response_handler.cpp

```diff
@@ -41,12 +41,15 @@ namespace iroha_cli {
     handler_map_[DATA_LOSS] = "DATA_LOSS";
   }
 
-  void GrpcResponseHandler::handle(
+  bool GrpcResponseHandler::handle(
       CliClient::Response<CliClient::TxStatus> response) {
     if (response.status.ok()) {
       tx_handler_.handle(response.answer);
+        return true;
```

Comment:
Please, fix indents, we use 2 spaces like here (https://google.github.io/styleguide/cppguide.html).